src/main/resources/api/service_openapi.yaml

```diff
@@ -114,6 +118,8 @@ paths:
           $ref: '#/components/responses/BadRequest'
         '403':
           $ref: '#/components/responses/PermissionDenied'
+        '404':
```

Comment:
aside: I wish OpenAPI gave us a way to avoid re-declaring the map between error and type everywhere.